examples/units_and_coordinates/STEREOSECCHI_starfield.py

```diff
@@ -0,0 +1,83 @@
+# -*- coding: utf-8 -*-
+"""
+===========================================================
+Identifying stars in a STEREO/SECCHI COR2 coronagraph image
+===========================================================
+
+How to use the Vizier star catalog to identify bright points in a
+STEREO/SECCHI COR 2 image as stars. As a bonus, we also identify Mars.
+"""
+import matplotlib.pyplot as plt
+
+from astroquery.vizier import Vizier
```

Comment:
You need to explain that astroquery may need to be installed.

Reply:
I think we will need to add it to the deps somewhere.

Reply:
I have two examples that need it.

Reply:
To be clear, I was asking for a comment to be added so that the reader understands that astroquery needs to be installed for the example to work.  (I don't know whether or not astroquery is already installed for our doc build.)

Reply:
Added a comment